### DISCUSSION TITLE
The openlayers don't render tiles on build

### DISCUSSION THREAD

**Discussion by JAEHOON-1993** [vercel/next.js#42935] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:22 PDT 2022; root:xnu-8020.121.3~4/RELEASE_X86_64
    Binaries:
      Node: 14.21.1
      npm: 6.14.17
      Yarn: 1.22.19
      pnpm: N/A
    Relevant packages:
      next: 13.0.3
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0

### What browser are you using? (if relevant)

Chrome

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

I am using React 18, Next 13, OL 7.1.0

It works when I execute the code "yarn run dev" like this

![image](https://user-images.githubusercontent.com/40525201/201827031-afc09865-c73a-43a6-bee2-4278b6ff1661.png)

But when I deploy on Vercel which is Next.js fron-end server. The tile don't load like this

![image](https://user-images.githubusercontent.com/40525201/201827072-853ad9dc-0d0a-42cd-adfa-783d0b1b9520.png)

How can I solve it?

Link: https://stackoverflow.com/questions/74411056/openlayer-tile-dont-render-on-next-js-vercel-server
Link: https://stackoverflow.com/questions/74396848/openlayer-tiles-do-not-render-on-build

### Expected Behavior

The tiles renders on dev version, but don't render on build especially on Vercel

There are many people suffering from this issue

### Link to reproduction - Issues with a link to complete (but minimal) reproduction code will be addressed faster

https://github.com/openlayers/openlayers/issues/14295

### To Reproduce

1. Type "yarn run dev", the openlayers tiles will render
2. When you deploy your build to Vercel, the tiles don't render

**Comment by balazsorban44** [comment]
Please see https://github.com/openlayers/openlayers/issues/14295#issuecomment-1312406977 and https://github.com/openlayers/openlayers/issues/14295#issuecomment-1312523877

It's hard to guess where the problem is without a reproduction.

**Reply by JAEHOON-1993** [reply]
Okay! I will give you the minimal project for reproduction of this issue

**Comment by MeRahulAhire** [comment]
Have a look here https://github.com/openlayers/openlayers/issues/14295#issuecomment-1320509944 Seems like it work on next 12 but not on next 13

**Reply by lawrencejob** [reply]
From that discussion, it looks like it's swcMinify that's causing an issue inside OpenLayers, but we're not sure which library is to blame.

Edit: it's a bug in swc
